src/mixins/common-mixin.ts

```diff
@@ -68,4 +74,27 @@ export default class CommonMixin extends Vue {
     }
     return true
   }
+
+  /**
+   * Navigates to the dashboard page, optionally with a filing ID.
+   * @param identifier The identifier to include in the dashboard URL
+   * @param filingId The filing ID to include in the dashboard URL (optional, defaults to null)
+   */
+  protected navigateToDashboard (identifier: string, filingId: number | null = null): void {
```

Comment:
By the way, we usually don't bother with a union type that includes null. We just assume that any property/variable can be null and code accordingly.

Reply:
Updated